src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -0,0 +1,385 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component renders a breadcrumb bar. It adjusts to the available width
+by hiding breadcrumbs in a dropdown menu and emits an event when something
+is dropped on a creadcrumb.
+
+### Usage
+
+```vue
+<template>
+	<div>
+		<div class="container container__wide">
+			<Breadcrumbs @dropped="dropped">
+				<Breadcrumb v-for="crumb in breadcrumbs" :title="crumb.title" :path="crumb.path">
+			</Breadcrumbs>
+		</div>
+		<br />
+		<div class="container container__narrow">
+			<Breadcrumbs @dropped="dropped">
+				<Breadcrumb v-for="crumb in breadcrumbs" :title="crumb.title" :path="crumb.path">
+			</Breadcrumbs>
+		</div>
+		<br />
+		<div class="dragme" draggable="true">
+			<span>Drag me onto the breadcrumbs.</span>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	computed: {
+		breadcrumbs() {
```

Comment:
Do we want to show example that complicated?
I think I would be ok to not usea v-for for the docs, dev should understand how it works?

Reply:
Fine with me.